jbpm-services/jbpm-kie-services/pom.xml

```diff
@@ -106,6 +106,11 @@
       <artifactId>kie-soup-maven-support</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-api</artifactId>
```

Comment:
we can't use this dependency as it will cause cyclic dependency issue as kie server is built after JBPM

Reply:
Dependency removed.